deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Turn off command traces while dealing with
+# the private key
+set +x
+
+echo -e $SSH_KEY | base64 --decode > ~/.ssh/id_rsa
+chmod 600 ~/.ssh/id_rsa
+
+npm run deploy
```

Comment:
Sorry to be a burden! I guess we want `deploy` to run outside of the protected traces.. Like:

```shell
set +x

# SSH stuffs

set -x

# Deploy

set +x

# Delete SSH

set -x
```

Reply:
Does that make sense? That way our private stuff is hidden but the deploy log is still visible for debugging :)